src/sass/components/ATATPhoneInput.scss

```diff
@@ -0,0 +1,122 @@
+._atat-phone-field {
+  ._country-select {
+    padding-right: 0px;
+    max-width: 67px;
+
+    .v-input__control > {
+      .v-input__slot {
+        padding-top: 8px;
+        padding-right: 4px;
+        margin-bottom: 0px;
+      }
+    }
+
+    fieldset {
+      border-radius: 4px 0px 0px 4px;
+    }
+  }
+
+  ._phone-number-input {
+    max-width: 351px;
+
+    &.v-text-field--outlined.v-input--is-focused {
+      fieldset {
+        border-left-width: 2px;
+        border-width: 2px;
+        border-color: $primary !important;
+      }
+    }
+
+    fieldset {
+      border-left-width: 0px;
+      border-radius: 0px 4px 4px 0px;
+    }
+
+    .v-text-field__prefix {
+      padding-left: 4px;
+      color: $base-darkest;
+    }
+  }
+
+  .v-menu__content.theme--light.menuable__content__active {
+    padding-top: 0px;
+    min-width: 416px !important;
+    width: 420px;
+
+    ._dropdown-text-field + ._suggested {
+      .v-list-item__content {
+        overflow: visible;
+        //margin-top: 26px;
+        position: relative;
+
+        &:before {
+          content: "SUGGESTED";
+          color: $base;
+          position: absolute;
+          top: -33px;
+          font-size: 12px;
+          font-weight: bold;
+        }
+      }
+    }
+
+    ._country-list {
+      padding-top: 6px;
+      padding-bottom: 6px
+    }
+
+    ._suggested {
+      + ._suggested {
+        border-bottom: 1px solid $base-lighter;
+      }
+    }
+
+    .v-select-list {
+      padding-top: 0px;
+      position: relative;
+    }
+
+    ._dropdown-text-field {
+      background-color: white;
+      position: sticky;
+      height: 48px;
+      width: 100%;
+      top: 0;
+      z-index: 1;
+      padding-top: 0px;
+      padding-right: 12px;
+      padding-left: 12px;
+      margin-top: 0;
+      margin-bottom: 30px;
+      color: $primary;
+      border-bottom: 1px solid $base-lighter;
+
+      .v-input__icon {
+        &.v-input__icon--append {
+          i.v-icon {
+            transform: none !important;
+            color: $primary !important;
+          }
+        }
+      }
+
+      &.v-input--is-focused {
+        .v-input__icon {
+          &.v-input__icon--append {
+            i.v-icon {
+              transform: none !important;
+              color: $primary !important;
+            }
+          }
+        }
+
+      }
+
+      :after, :before {
```

Comment:
these probably need `&` ... so `&:after, &:before` -- without the `&` it doesn't attach itself to anything

Reply:
![Screen Shot 2022-03-29 at 9 31 52 AM](https://user-images.githubusercontent.com/79282243/160661034-c0168ca3-3753-46e8-bf5e-b923b4e3fefa.png)
adding the & in front of them adds this border under the text input